.travis.yml

```diff
@@ -1,15 +1,29 @@
 language: python
 python:
   - 3.6
+sudo: required
+addons:
+  chrome: stable
+services:
+  - xvfb
 cache: pip
 install:
   - pip install -r requirements.txt -r requirements-dev.txt
+before_script:
+  - wget "https://chromedriver.storage.googleapis.com/77.0.3865.40/chromedriver_linux64.zip"
+  - unzip chromedriver_linux64.zip
+  - sudo mv chromedriver /usr/local/bin
+  - "export DISPLAY=:99.0"
 script:
   - python -m coverage run --source rfhub2 -m unittest tests
   - black --check -t py36 rfhub2 tests
-  - python publish_pypi.py
+  - python setup.py install
+  - pip install -r tests/requirements-robot.txt
```

Comment:
If we could add virtualenv here, we would ensure that package is not missing any dependency

Reply:
I will find a way :D